packages/browser-service/README.md

```diff
@@ -0,0 +1,78 @@
+# Browser service
+
+This package provides the BrowserService class for initializing and managing a Playwright browser context, network configurations, and wallets (EOA or WalletConnect). It facilitates automated testing of interactions with Web3 wallets (e.g., Metamask) and offers convenient methods for network switching, connecting to a fork node, etc.
+
+# Installation
+```
+npm install browser-service
+# or
+yarn add browser-service
+```
+
+# Quick Start
+
+```typescript
+import { test, expect } from '@playwright/test';
+import { BrowserService } from 'browser-service';
+
+test.describe('Example usage of BrowserService', () => {
+  let browserService: BrowserService;
+
+  test.beforeAll(async () => {
```

Comment:
Could you use already defined objects in example as it is in `initBrowserService` to avoid miss understanding related to objects?